tracer/src/Datadog.Trace/SpanKinds.cs

```diff
@@ -35,5 +35,11 @@ public static class SpanKinds
         /// </summary>
         /// <seealso cref="Tags.SpanKind"/>
         public const string Consumer = "consumer";
+
+        /// <summary>
+        /// A span that represents an internal operation within an application.
+        /// </summary>
+        /// <seealso cref="Tags.SpanKind"/>
+        public const string Internal = "internal";
```

Comment:
Is this a standardized value across tracers?

Reply:
Resolving because Lucas suggested this value and I assume he's always correct 😛

Reply:
> I assume he's always correct

Oh no, please don't do that 😓.

But in this case, `span.kind` is also used in OpenTracing/OpenTelemetry, where they define the other values in this file plus `internal` (the default is not specified). I'm ok with adding the constant here, but after a short discussion in the RFC for this feature, I maybe we should not the `span.kind` tag at all. (but adding it won't hurt, either).